pkgs/sagemath-brial/sage

```diff
@@ -0,0 +1 @@
+../../src/sage
```

Comment:
Can you please update https://github.com/sagemath/sage/blob/b7c1c8fe5cd0235a8c28d41d18a16dcbb9f10391/.vscode/settings.json#L10-L17 if you add new packages / symlinks to `src`. Thanks!

Reply:
Ah, thanks, this is of course already documented in our developer's manual, but I forgot about it.

Reply:
Updated in 8a2be7d; please test

Reply:
Thanks

Reply:
This commit is now #36511